src/CMakeLists.txt

```diff
@@ -1042,7 +1038,9 @@ set(CPACK_COMPONENTS_ALL DTApplication DTDebugSymbols DTDocuments)
 
 if (USE_LIBRAW)
   set(LIBRAW_PATH "${CMAKE_CURRENT_SOURCE_DIR}/external/LibRaw" CACHE STRING "Relative path to libraw directory (default=CMAKE_CURRENT_SOURCE_DIR)")
-  set(ENABLE_EXAMPLES OFF CACHE BOOL "")
+  if(NOT USE_OPENMP)
+    set(ENABLE_OPENMP OFF CACHE BOOL "")
+  endif()
   set(DT_SUPPORTED_EXTENSIONS ${DT_SUPPORTED_EXTENSIONS} libraw CACHE INTERNAL "")
```

Comment:
Tried it out and it compiles with OPENMP support.  Tested it and it works fine.  Pushed the branch to my repository and tried to trigger a CI run, but didn't figure out how to make it work.

@TurboGit looks good to me